packages/sql/src/index.js

```diff
@@ -1,6 +1,5 @@
-'use strict'
-
 const EventEmitter = require('events')
+const { Sql } = require('fix-esm').require('sql-ts')
```

Comment:
adding `fix-esm` worries me.

Maybe can we compile the CJS at GitHub Actions?

Like this: https://github.com/apify/got-cjs/blob/main/.github/workflows/publish.yml